package.json

```diff
@@ -33,7 +33,8 @@
     "express-prismic": "^1.1.1",
     "graphql": "^0.6.0",
     "lodash": "^4.13.1",
```

Comment:
but then it would be Babel's responsibility to maintain that dep? Pretty sure you don't have to have it in your package.json to have Babel working.